Directions.md

```diff
@@ -33,3 +33,56 @@ better. Also make sure to slide the wings on the skewer as they tend to stick to
 9) Once everything is cooked, you can put the skewers between the slices of bread and pull the skewer out. This will leave the wings between the slices of bread and they are ready to be served. 
 
 Bon apetit!
+
+**Important Note:** All the recipes listed below for the salads are for 1 serving. 
+
+### Onion Salad
+
+1) Chop 1/2 yellow or white onion very fine (papper thin pieces)
+2) Chop 1/4 bunch of the Italian parsley very fine
+3) Add the chopped onion and parsley into a salad bowl
+4) Add 1 table spoons of Sumac (or more if desired)
```

Comment:
Lets get some additional specifications on the Sumac, as this can be confusing.  There is a Native American Sumac.  I recommend either a hyperlink or more descriptive spec for the Sumac, as I believe it could be different in Turkey.